### PR TITLE
e2e: Add run --memory-swap test

### PR DESCRIPTION
**Pull request by sstosh**
There is not e2e/system test of --memory-swap option.

Signed-off-by: Toshiki Sonoda <sonoda.toshiki@fujitsu.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks!

@giuseppe @edsantiago PTAL

**Review by giuseppe — APPROVED** [review]
/lgtm
/hold

File: test/e2e/run_memory_test.go [hunk 1]
```diff
@@ -66,6 +66,24 @@ var _ = Describe("Podman run memory", func() {
...
 	})
 
+	It("podman run memory-swap test", func() {
+		var (
+			session *PodmanSessionIntegration
+			expect  string
+		)
+
+		if CGROUPSV2 {
+			session = podmanTest.Podman([]string{"run", "--memory=20m", "--memory-swap=30M", "--net=none", ALPINE, "sh", "-c", "cat /sys/fs/cgroup/$(sed -e 's|0::||' < /proc/self/cgroup)/memory.swap.max"})
```
Comment (edsantiago) [inline test/e2e/run_memory_test.go:76]: Just curious: is the `sed` thing really necessary? Are there any circumstances in which it won't just be `/`?

File: test/e2e/run_memory_test.go [hunk 2]
```diff
@@ -66,6 +66,24 @@ var _ = Describe("Podman run memory", func() {
...
+			expect = "10485760"
```
Comment (edsantiago) [inline test/e2e/run_memory_test.go:77]: People not intimately familiar with cgroups v2 might appreciate a comment like `// 30m - 20m` as a reminder that cgroups v2 stores the delta, not the total. But this is not a showstopper. LGTM.

**Review by edsantiago — APPROVED** [review]

**Pull request merged**